cmd/dicode/init.go

```
@@ -363,21 +376,61 @@ func writeGitignore(path, line string) error {
+	dbPath := filepath.Join(dataDir, dataDBName)
+	if _, err := os.Stat(dbPath); err != nil {
+		return ""
+	}
+	handle, err := sql.Open("sqlite", "file:"+dbPath+"?mode=ro")
+	if err != nil {
+		return ""
+	}
+	defer func() { _ = handle.Close() }()
+
+	var value string
+	if err := handle.QueryRow(
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).QueryRow must not be called. use (*database/sql.DB).QueryRowContext

(noctx)